### ISSUE TITLE
Getting compile errors

### ISSUE THREAD

**Issue by ghxstweb** [ghseeli/latex2blender#23]
It installed the texlive-bin (arch distro) and the dvisvgm binaries accordingly, although I still get these issues that prevent compiling:
![image](https://github.com/user-attachments/assets/aa2f7c6f-4ebd-4ad3-8e85-666f72e3164c)

any idea s to how I can address them? Thanks!

**Comment by ghxstweb** [comment]
I went ahead an used `sudo pacman -S texlive` through the whoel installation and got a different set of error. Is it now a problem with dvisvgm?

![image](https://github.com/user-attachments/assets/08bc61bf-8afd-4a14-b7e2-c939f5b07fa3)

**Comment by peterkj1** [comment]
Yes, it is likely a problem with dvisvgm. To diagnose the problem, it might be helpful to try running dvisvgm in the command line and see if you get any error messages. For example, you could create a random .dvi file (call it say test.dvi) and run the command `dvisvgm --no-fonts test.dvi` in your terminal. See also https://dvisvgm.de for dvisvgm documentation.
